daemon/mgr/container_validation.go

```diff
@@ -91,6 +93,46 @@ func (mgr *ContainerManager) validateConfig(c *Container, update bool) ([]string
 	return warnings, nil
 }
 
+// validateDiskQuota is used to validate disk quota config
+func (mgr *ContainerManager) validateDiskQuota(config *types.ContainerCreateConfig) error {
+	if config == nil {
+		return errors.Errorf("invalid request, create config is nil")
+	}
+
+	if config.DiskQuota == nil {
+		if config.QuotaID != "" && config.QuotaID != "0" {
+			return errors.Wrap(errInvalidDiskQuota, "set QuotaID without DiskQuota")
+		}
+		return nil
+	}
+
+	quota := config.DiskQuota
+	if len(quota) > 1 && config.QuotaID != "" {
```

Comment:
this function has the same function with `ParseQuotaID`. could we consider to reuse it?

Reply:
ParseQuotaID just do a simple check for cli, validateDiskQuota will do more. so no need to reuse.